lib/src/io_web_auth.dart

```diff
@@ -12,9 +12,8 @@ class IoWebAuth implements BaseWebAuth {
       required String redirectUrl,
       Map<String, dynamic>? opts}) async {
     return await FlutterWebAuth2.authenticate(
-      callbackUrlScheme: callbackUrlScheme,
-      url: url,
-      preferEphemeral: (opts?['preferEphemeral'] == true),
-    );
+        callbackUrlScheme: callbackUrlScheme,
+        url: url,
+        options: FlutterWebAuth2Options.fromJson(opts ?? {}));
```

Comment:
`FlutterWebAuth2Options.fromJson(...)` needs more work. I am getting
```
_TypeError (type 'Null' is not a subtype of type 'bool')
```
for the `preferEphemeral` key, because an empty JSON is used.

As a workaround, I did
```
      options: FlutterWebAuth2Options(
          preferEphemeral: (opts?['preferEphemeral'] == true)),
```

Reply:
Fixed upstream